libraries/plugins/market_history/market_history_plugin.cpp

```diff
@@ -174,17 +174,25 @@ struct operation_process_fill_order
              db.modify( *itr, [&]( bucket_object& b ){
                   b.base_volume += trade_price.base.amount;
                   b.quote_volume += trade_price.quote.amount;
-                  b.close_base = trade_price.base.amount;
-                  b.close_quote = trade_price.quote.amount;
-                  if( b.high() < trade_price ) 
+                  if(trade_price.base.amount >= 100 && trade_price.quote.amount >= 100)
```

Comment:
Although this is okay for most "popular" trading pairs, it's not ideal to be applied to all pairs, especially UIA's.

I think it's better that we record the data that how much difference is caused by rounding when matching orders, or simply record the "fair" price there, then process the data here.